.gitignore

```diff
@@ -8,8 +8,13 @@
 /backend/.benchmarks
 /backend/keys/nyrkio.pem
```

Comment:
Actually anything in keys/ should never be in git!

https://github.com/nyrkio/nyrkio/blob/88adb2f76b15328bdb0dce5b86179d26cb7a60dc/.github/workflows/deploy-prod.yml#L51